prov/gni/src/gnix_dom.c

```diff
@@ -427,8 +433,8 @@ gnix_domain_ops_open(struct fid *fid, const char *ops_name, uint64_t flags,
 	return ret;
 }
 
-int gnix_domain_open(struct fid_fabric *fabric, struct fi_info *info,
-		     struct fid_domain **dom, void *context)
+DIRECT_FN int gnix_domain_open(struct fid_fabric *fabric, struct fi_info *info,
```

Comment:
So, all of the GNI API functions which are assigned in the function table should not be declared as STATIC otherwise they should be declared as STATIC?